storage/src/device.rs

```diff
@@ -402,10 +402,18 @@ pub trait BlobChunkInfo: Any + Sync + Send {
 }
 
 /// An enumeration to encapsulate different [BlobChunkInfo] implementations for [BlobIoDesc].
+/// This helps to feed unified IO description to storage subsystem from both rafs v6 and v5 since
+/// rafs v6 have a different ChunkInfo definition on bootstrap.
 #[derive(Clone)]
 pub enum BlobIoChunk {
+    // For rafs v6 to pass chunk info to storage module.
+    // (blob_index, chunk_index)
     Address(u32, u32),
+    // TODO: `Address` is is always converted to `Base` for v6, it is a little confusing.
```

Comment:
The rafs fuse has bootstrap, so `ChunkInfo` is available.
But for blobfs/fscache, they only know about requested address range(start, len).
So `Address` is designed for blobfs/fscache, and the storage subsystem will translate `Address` into `ChunkInfo`. 
The "Address" is designed to support virtiofs/blobfs/fscache, which have no concept of "ChunkInfo" at all.